routes/index.js

```diff
@@ -0,0 +1,7 @@
+const router = require('express').Router()
+
```

Comment:
If you think it will be more readable to require the router, then have a blank line, and then require all of the routes though, maybe we do